cargo-typify/schema_examples/basic.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Realizing as I'm hoping this draft that I can probably use the examples at `typify/tests/schemas`

So I will likely clean this up!